### DISCUSSION TITLE
What about Redis 3 ？ I am using Redis 3.

### DISCUSSION THREAD

**Discussion by yxw2014** [redis/go-redis#2311] [Q&A]
If you are using Redis 6, install go-redis/v8:
go get github.com/go-redis/redis/v8

If you are using Redis 7, install go-redis/v9:
go get github.com/go-redis/redis/v9

------------------------------------------------------

**What about Redis 3 ?   I am using Redis 3.😡**

**Comment by vmihailenco** [comment]
Such old Redis versions are not tested and supported. The client may still work though.
